lib/membrane_rtc_engine/endpoints/webrtc/rtp_connection_allocator.ex

```diff
@@ -155,7 +174,10 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.RTPConnectionAllocator do
       pid: pid,
       current_allocation: bandwidth,
       target_allocation: nil,
-      negotiable?: negotiable?
+      negotiable?: negotiable?,
+      last_estimation_ts: get_timestamp(),
+      bits_since_last_estimation: 0,
+      estimated_sender_rate: bandwidth
```

Comment:
We can keep this name but it's just track_bitrate and I believe it's easier to read code when we have conditions like `if track_bitrate < bandwidth, do: ...`